main.rb

```diff
@@ -0,0 +1,39 @@
+require './classes/app'
```

Comment:
- [ ] Your app wouldn't run from my end because of the way you imported this file, I changed it to `require_relative 'app'` and ran `ruby main.rb` in the terminal, and everything worked fine. Please double check and make sure everything works fine from everyones machine. It will also with nice to update the `readme file` with explicit steps to clone and run your app locally, especially the command to run the app as this will be of help to non technical users.